templates/method.mustache

```diff
@@ -33,10 +33,11 @@ $domain?: string
         {{/isPathParameter}}
     {{/parameters}}
     
-    if(parameters.$queryParameters) {
-        Object.keys(parameters.$queryParameters).forEach(function(parameterName) {
-            queryParameters[parameterName] = parameters.$queryParameters[parameterName];
-        });
+    if (parameters.$queryParameters != undefined) {
```

Comment:
Lets use !== instead to be as precise as possible. I.e. this would then look like:

`parameters.$queryParameters !== undefined && parameters.$queryParameters !== null`

Reply:
You decide, but: 🤓 

I used `!= undefined` intentionally, because it's idiomatic JavaScript to check *exactly* for not `undefined` and not `null`. In fact, this is the only situation where I use such `==` / `!=` expressions, in all other cases I use `===`, too.

See for example here: <https://basarat.gitbooks.io/typescript/docs/javascript/null-undefined.html>

Reply:
Hehe yes, I just fear that someone will believe using != is an error and replace it with !== without thinking about it, so making it explicit would be (slightly) safer :)

Reply:
What about

```javascript
if (parameters.$queryParameters != undefined) { // 🕶 Trust me, I know what I'm doing
  // ...
```

Just kidding 🤡